### PR TITLE
Update Network Protection for macOS docs

### PR DESCRIPTION
**Pull request by verndrade**
Add updates now that Network Protection for Mac is GA

### CODE REVIEW

File: microsoft-365/security/defender-endpoint/network-protection-macos.md [hunk 1]
```diff
@@ -44,19 +44,28 @@ Network protection expands the scope of Microsoft 365 Defender [SmartScreen](/wi
...
 
-## New and updated capabilities
+## Current capabilities
 
-- You can run your corporate VPN in tandem or "side by side" with network protection. Currently, no VPN conflicts are identified. If you do experience conflicts, you can provide feedback through the feedback channel listed at the bottom of this page.
-  - Web content filtering is supported with network protection for macOS.
-  - If network protection is configured and active on the device, web content filtering (WCF) policies created in the MDEP Portal are respected in browsers, including Chromium Microsoft Edge for macOS. Web content filtering in Microsoft Edge on Mac currently requires network protection; other E5 features, such as Microsoft Defender for Cloud Apps or Custom Indicators, currently also require network protection.
+- Custom Indicators of Compromise on Domains and IPs
+- Web Content Filtering support -
+  - Block website categories scoped to device groups through policies created in the MDEP Portal.
```
Comment (JohanFreelancer9) [inline microsoft-365/security/defender-endpoint/network-protection-macos.md]:
```suggestion
- Custom Indicators of Compromise on Domains and IPs.
- Web Content Filtering support:
  - Block website categories scoped to device groups through policies created in the MDEP portal.
```

File: microsoft-365/security/defender-endpoint/network-protection-macos.md [hunk 2]
```diff
@@ -44,19 +44,28 @@ Network protection expands the scope of Microsoft 365 Defender [SmartScreen](/wi
...
+  - Policies are applied to browsers, including Chromium Microsoft Edge for macOS. 
+- Advanced Hunting - Network Events will be reflected in the Machine Timeline, and queryable in Advanced Hunting to aid security investigations
```
Comment (JohanFreelancer9) [inline microsoft-365/security/defender-endpoint/network-protection-macos.md]:
```suggestion
- Advanced Hunting - Network Events will be reflected in the Machine Timeline, and queryable in Advanced Hunting to aid security investigations.
```

File: microsoft-365/security/defender-endpoint/network-protection-macos.md [hunk 3]
```diff
@@ -44,19 +44,28 @@ Network protection expands the scope of Microsoft 365 Defender [SmartScreen](/wi
 
 ## Availability
 
-Network Protection for macOS will soon be available for all Microsoft Defender for Endpoint onboarded macOS devices which meet the minimum requirements. Microsoft will begin incrementally rolling out the functionality for all macOS devices to enable Network Protection on 1/31/2023 with target completion, subject to change, in May 2023. When this feature rolls to production, all of your currently configured Network Protection and Web Threat Protection policies will be enforced on macOS devices where Network Protection is configured for block mode.
+Network Protection for macOS is now available for all Microsoft Defender for Endpoint onboarded macOS devices which meet the minimum requirements. All of your currently configured Network Protection and Web Threat Protection policies will be enforced on macOS devices where Network Protection is configured for block mode.
```
Comment (JohanFreelancer9) [inline microsoft-365/security/defender-endpoint/network-protection-macos.md]:
```suggestion
Network Protection for macOS is now available for all Microsoft Defender for Endpoint onboarded macOS devices that meet the minimum requirements. All of your currently configured Network Protection and Web Threat Protection policies will be enforced on macOS devices where Network Protection is configured for block mode.
```

File: microsoft-365/security/defender-endpoint/network-protection-macos.md [hunk 4]
```diff
@@ -44,19 +44,28 @@ Network protection expands the scope of Microsoft 365 Defender [SmartScreen](/wi
...
 
-To prepare for the macOS network protection rollout, we recommend the following:
+To rollout Network Protection for macOS, we recommend the following:
```
Comment (JohanFreelancer9) [inline microsoft-365/security/defender-endpoint/network-protection-macos.md]:
```suggestion
To roll out Network Protection for macOS, we recommend the following:
```

File: microsoft-365/security/defender-endpoint/network-protection-macos.md [hunk 5]
```diff
@@ -44,19 +44,28 @@ Network protection expands the scope of Microsoft 365 Defender [SmartScreen](/wi
...
 
-- For Network Protection for macOS to be active on your devices, Network Protection must be enabled by your organization. We suggest deploying the audit or block mode policy to a small set of devices and verify there are no issues or broken workstreams before gradually deploying to a larger set of devices.
-- Verify the Network Protection configuration on your macOS devices is set to the desired state.
-- Understand the impact of your Web Threat Protection, Custom Indicators of Compromise, Web Content Filtering, and MDA Endpoint Enforcement policies which target those macOS devices where Network Protection is in Block mode.
+- Create a device group for a small set of device that you can use to test Network Protection.
+- Evaluate the impact of Web Threat Protection, Custom Indicators of Compromise, Web Content Filtering, and Microsoft Defender for Cloud Apps enforcement policies which target those macOS devices where Network Protection is in Block mode.
```
Comment (JohanFreelancer9) [inline microsoft-365/security/defender-endpoint/network-protection-macos.md]:
```suggestion
- Create a device group for a small set of devices that you can use to test Network Protection.
- Evaluate the impact of Web Threat Protection, Custom Indicators of Compromise, Web Content Filtering, and Microsoft Defender for Cloud Apps enforcement policies that target those macOS devices where Network Protection is in Block mode.
```

File: microsoft-365/security/defender-endpoint/network-protection-macos.md [hunk 6]
```diff
@@ -44,19 +44,28 @@ Network protection expands the scope of Microsoft 365 Defender [SmartScreen](/wi
...
 
-- You can run your corporate VPN in tandem or "side by side" with network protection. Currently, no VPN conflicts are identified. If you do experience conflicts, you can provide feedback through the feedback channel listed at the bottom of this page.
-  - Web content filtering is supported with network protection for macOS.
-  - If network protection is configured and active on the device, web content filtering (WCF) policies created in the MDEP Portal are respected in browsers, including Chromium Microsoft Edge for macOS. Web content filtering in Microsoft Edge on Mac currently requires network protection; other E5 features, such as Microsoft Defender for Cloud Apps or Custom Indicators, currently also require network protection.
+- Custom Indicators of Compromise on Domains and IPs
+- Web Content Filtering support -
+  - Block website categories scoped to device groups through policies created in the MDEP Portal.
+  - Policies are applied to browsers, including Chromium Microsoft Edge for macOS. 
+- Advanced Hunting - Network Events will be reflected in the Machine Timeline, and queryable in Advanced Hunting to aid security investigations
+- Microsoft Defender for Cloud Apps
```
Comment (JohanFreelancer9) [inline microsoft-365/security/defender-endpoint/network-protection-macos.md]:
```suggestion
- Microsoft Defender for Cloud Apps:
```

File: microsoft-365/security/defender-endpoint/network-protection-macos.md [hunk 7]
```diff
@@ -44,19 +44,28 @@ Network protection expands the scope of Microsoft 365 Defender [SmartScreen](/wi
...
+  - Shadow IT discovery - Identify which apps are being used in your organization.
+  - Blocking Applications - Block entire applications (e.g Slack, Facebook) from being used in your organization
```
Comment (JohanFreelancer9) [inline microsoft-365/security/defender-endpoint/network-protection-macos.md]:
```suggestion
  - Block applications - Block entire applications (such as Slack and Facebook) from being used in your organization.
```

File: microsoft-365/security/defender-endpoint/network-protection-macos.md [hunk 8]
```diff
@@ -44,19 +44,28 @@ Network protection expands the scope of Microsoft 365 Defender [SmartScreen](/wi
...
+- Corporate VPN in tandem or "side by side" with Network Protection.
```
Comment (JohanFreelancer9) [inline microsoft-365/security/defender-endpoint/network-protection-macos.md]:
```suggestion
- Corporate VPN in tandem or side-by-side with Network Protection: 
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - @verndrade, kindly commit the suggested changes. Many thanks.

**Comment by verndrade** [comment]
> Copy review - @verndrade, kindly commit the suggested changes. Many thanks.

Thanks for the review!

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by yogkumgit** [comment]
@Dansimp Please merge this PR. Thanks

**Pull request merged**